observe/resource_monitor_v2.go

```diff
@@ -952,6 +962,9 @@ func newMonitorV2Input(data *schema.ResourceData) (input *gql.MonitorV2Input, di
 	if v, ok := data.GetOk("description"); ok {
 		input.Description = stringPtr(v.(string))
 	}
+	if v, ok := data.GetOk("disabled"); ok {
+		input.Disabled = boolPtr(v.(bool))
+	}
```

Comment:
Due to a terraform sdk limitation, if `disabled` is `false`, then the `ok` here will always be `false` (and therefore `input.Disabled` will be `nil`/omitted instead of being `false`). So assuming that the API treats `disabled` being omitted in the input as "leave unchanged", this would mean it's impossible re-enable a monitor through terraform after disabling it. Would probably suggest doing the following instead:
```suggestion
	input.Disabled = boolPtr(data.Get("disabled").(bool))
```

Reply:
Actually it looks like the API treats `disabled` being omitted as `false`, so approving. But may want to make this change anyway